packages/babel-plugin-transform-parameters/src/rest.js

```diff
@@ -295,12 +306,13 @@ export default function convertFunctionRest(path) {
     state.candidates.map(({ path }) => path),
   );
 
-  const start = t.numericLiteral(node.params.length);
+  const paramsCount = getParamsCount(node);
```

Comment:
It might be better to hoist this const to above the `const state`, so that you can just re-use the `paramsCount` as the offset in state.

Reply:
Okay, PR edited!